components/atom/label/src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const AtomLabel = ({name, text, optional}) => (
+  <label htmlFor={name} className='sui-AtomLabel'>
+    {text}
+    <span className='sui-AtomLabel-optional'>
+      {optional}
+    </span>
+  </label>
+)
+
+AtomLabel.displayName = 'AtomLabel'
+
+AtomLabel.defaultProps = {
+  optional: '(optional)'
+}
+
+AtomLabel.propTypes = {
+  name: PropTypes.string.isRequired,
```

Comment:
please add the description to the components like done in other components https://github.com/SUI-Components/sui-components/blob/master/components/atom/button/src/index.js#L88